uavcan/equipment/ahrs/1002.RawIMU.uavcan

```diff
@@ -0,0 +1,23 @@
+#
+# Raw IMU data with timestamps
+#
+
+
+
+#
+# rad/sec
+#
+uavcan.Timestamp gyro_timestamp
+float16[3] gyro_raw
+float16[3] gyro_integral
+void4
+float16 gyro_integral_dt
```

Comment:
I noticed that you removed the field `float16[<=9] gyro_covariance`. If you chose to remove it, you should also have removed the alignment field `void4`. I suggest to keep the covariance field though, probably in the form of a single 6x6 matrix at the end of the message. In that case it will be possible to convey covariance information between acceleration and angular velocity measurements. The right syntax is `float16[<=36] gyro_accel_covariance`. See http://uavcan.org/Specification/5._Application_level_conventions/#matrix-representation.